.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+install: "pip install -r requirements.txt"
+
+before_script: ablog build
```

Comment:
Yeah, I think this line needs to be deleted. it seems to be why travis is failing to build.